mediated_transfer.rst

```diff
@@ -0,0 +1,71 @@
+Mediated transfers
+##################
+
+Overview
+========
+
+The protocol supports mediated transfers. Nodes may use them to send payments.
```

Comment:
very good idea to outsource this chapter. Here, why you don't have `term: Mediated Transfer` like the lines below?